arm-recoveryservicessiterecovery/2016-08-10/swagger/service.json

```diff
@@ -0,0 +1,14114 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2016-08-10",
+    "title": "SiteRecoveryManagementClient"
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/Subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.RecoveryServices/vaults/{resourceName}/replicationFabrics/{fabricName}/replicationProtectionContainers/{protectionContainerName}/replicationProtectedItems": {
+      "get": {
+        "tags": [
+          "ReplicationProtectedItems"
+        ],
+        "summary": "Gets the list of Replication protected items.",
+        "description": "Gets the list of ASR replication protected items in the protection container.",
+        "operationId": "ReplicationProtectedItems_ListByReplicationProtectionContainers",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersion"
+          },
+          {
+            "$ref": "#/parameters/ResourceName"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupName"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionId"
+          },
+          {
+            "name": "fabricName",
```

Comment:
Ok, up to you, my suggestion was mostly for your convenience, so you didn't need to repeat and modify everywhere you needed an update in the swagger.